### PR TITLE
*/tests: remove ineffectual assignment in tests

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: AndreMouche

### What problem does this PR solve?
This PR removes ineffectual assignment in tests,then we can `make check` successfully
Part of https://github.com/pingcap/tidb/issues/22513

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: store/tikv/tests/snapshot_fail_test.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func (s *testSnapshotFailSuite) TestRetryPointGetResolveTS(c *C) {
 
 	txn, err := s.store.Begin()
 	c.Assert(err, IsNil)
-	err = txn.Set([]byte("k1"), []byte("v1"))
+	txn.Set([]byte("k1"), []byte("v1"))
```
Comment (sticnarf) [inline store/tikv/tests/snapshot_fail_test.go:209]: Emm... I thought the linter should complain that the returned error is not used...
I think a better idea is to add an assertion

**Review by sticnarf — APPROVED** [review]
LGTM

**Pull request merged**